packages/ui/src/helpers/authenticator/formFields/formFields.ts

```diff
@@ -35,11 +35,18 @@ export const getFormFields = (
   return formFields;
 };
 
+export const removeOrderKeys = (formFields: FormFieldsArray): FormFieldsArray =>
+  formFields.map((field) => {
+    const key = field[0];
+    const values = { ...field[1], order: undefined };
```

Comment:
Should we add a comment for why we are doing this?
```suggestion
    const values = { ...field[1], order: undefined }; // Drop order key to prevent passing to form field UI components 
```

Reply:
```suggestion
    // Drop order key to prevent passing to form field UI components
    const values = { ...field[1], order: undefined };
```